pommerman/forward_model.py

```diff
@@ -532,12 +533,13 @@ def in_view_range(position, v_row, v_col):
             bomb_blast_strengths, bomb_life = make_bomb_maps(agent.position)
             agent_obs['bomb_blast_strength'] = bomb_blast_strengths
             agent_obs['bomb_life'] = bomb_life
+            agent_obs['game_type'] = game_type.name
```

Comment:
'name' or 'value'? the latter will be an int and probably easier to use given it's an enum.

Reply:
yeah. that makes sense.